test/services/user/services/AdminUsers.test.js

```diff
@@ -333,6 +356,35 @@ describe('AdminTeachersService', () => {
 		expect(adminTeachersService).to.not.equal(undefined);
 	});
 
+	it('student can not administrate teachers', async () => {
+		const student = await testObjects.createTestUser({ roles: ['student'] });
+		const params = await testObjects.generateRequestParamsFromUser(student);
+		params.query = {};
+		try {
+			await adminTeachersService.find(params);
+			throw new Error('should have failed');
+		} catch (err) {
+			expect(err.message).to.not.equal('should have failed');
+			expect(err.message).to.equal(testGenericErrorMessage);
+			expect(err.code).to.equal(403);
+		}
+	});
+
+	it('teacher can not administrate teachers', async () => {
+		const teacher = await testObjects.createTestUser({ roles: ['teacher'] });
+		const params = await testObjects.generateRequestParamsFromUser(teacher);
+		params.query = {};
+		const result = await adminTeachersService.find(params);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['result' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5676825)